stock_release_channel_shipment_advice/models/stock_release_channel.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import Command, _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockReleaseChannel(models.Model):
+    _inherit = "stock.release.channel"
+
+    shipment_planning_method = fields.Selection(
```

Comment:
We could make the column required without trouble by altering the column definition into a post_init hook to declare the default value into the column definition.